fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -1555,7 +1555,7 @@ functionCallExpression
     : functionIdentifier
               LEFT_PAREN (
                   (DISTINCT|ALL)?
-                  arguments+=expression (COMMA arguments+=expression)*
+                  arguments+=argumentExpression (COMMA arguments+=argumentExpression)*
```

Comment:
1. namedExpression contains expression, so use namedExpression directly is ok
2. i think any other function should throw exception explicitly, such as abs(a as b)